src/common/event-emitter/contextual-ee.js

```diff
@@ -21,12 +23,19 @@ if (nr.ee) {
 
 export { globalInstance as ee }
 
-function EventContext () {}
+function EventContext() { }
 
-function ee (old, debugId) {
+function ee(old, debugId) {
   var handlers = {}
   var bufferGroupMap = {}
   var emitters = {}
+  // backlog must be isolated if multiple agents can run on page. MFE loader packages sets this to true. only check this for feature emitters (length = 16)
+  var isolatedBacklog = false 
+  try{
+    isolatedBacklog = debugId.length !== 16 ? false : getRuntime(debugId).isolatedBacklog 
```

Comment:
Somewhat of a magic number here. Is this corresponding to generateHexString for agentId?

Reply:
yes, thats correct, checking if the EE's id is an agentIdentifier.  Some of our tests for instance do not namespace the EE with an agentID at all and just use the parent (which has no id)

Reply:
https://issues.newrelic.com/browse/NEWRELIC-6479